src/components/packages/Package.js

```diff
@@ -129,7 +129,7 @@ class Package extends Component {
 
         return <div className="fs-selected-pricing-license-quantity">
             {selectedPricing.sitesLabel()}
-            { ! planPackage.is_free_plan && <Tooltip>
+            { ! planPackage.is_free_plan && <Tooltip direction={this.getTooltipDirection()}>
```

Comment:
Please don't use the short-circuit (&&) for conditional rendering. Kindly use the ternary operator.

```jsx
{ !planPackage.is_free_plan ? (
  <RenderThisComponent />
) : null}
```

Please read about the [reasoning here](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx).